low-level/framework/rabbitmq/rabbitmq_egress_processor.py

```diff
@@ -26,6 +26,10 @@
 
 import pika
 
+from cortx.utils.message_bus import MessageProducer
```

Comment:
Can we rename these files like below. or put your proposals
rabbitmq_egress_processor.py -> egress_processor.py
rabbitmq_egress_accumulated_msgs_processor.py  -> egress_accumulated_msgs_processor.py 
rabbitmq_ingress_processor.py  -> ingress_processor.py

Reply:
That will be changed in remove RabbitMQ references [[EOS-16335](https://jts.seagate.com/browse/EOS-16355)]